portal-bridge/src/bridge/era1.rs

```diff
@@ -246,13 +250,20 @@ impl Era1Bridge {
                 panic!("Failed to get epoch from era1 file: {e}");
             }
         };
-        let epoch_acc = match self.get_epoch_acc(epoch_index).await {
+        let (_, epoch_acc) = match lookup_epoch_acc(
```

Comment:
It doesn't seem like we're using the `epoch_hash` that's being returned as the first value anymore, so you could just update this util fn to return the `epoch_acc`